src/Symfony/Component/DependencyInjection/Loader/YamlFileLoader.php

```diff
@@ -194,7 +194,7 @@ private function parseDefinitions($content, $file)
                 }
             }
         } else {
-            $defaults = array();
+            $defaults = array('public' => true);
```

Comment:
Actually, would you prefer using the same way as you did for classic definitions, i.e:

```php
$public = isset($service['public']) ? $service['public'] : (isset($defaults['public']) ? $defaults['public'] : true);
```

rather than adding this in defaults?